Sources/Reactive.swift

```diff
@@ -0,0 +1,26 @@
+/// Describes a class which has been extended with reactive elements.
+///
+/// - note: `ExtendedForReactiveness` is only intended for extensions to types
+///         that are not owned by the module. Non-conforming types may carry
+///         first-party reactive elements.
+public protocol ExtendedForReactiveness: class {}
+
+extension ExtendedForReactiveness {
+	/// A proxy which exposes the reactivity of `self`.
+	public var rac: Reactive<Self> {
+		return Reactive(self)
+	}
+}
+
+// A proxy which exposes the reactivity of the Reactant type.
+public struct Reactive<Reactant: ExtendedForReactiveness> {
```

Comment:
I think `Reactant` should be called `Base` instead. RxSwift did that, and I think it's the right call because it matches what the Swift standard library does for similar types. (c.f. `LazySequence`)

Also, I don't think `Reactant` needs to conform to `ExtendedForReactiveness`.